samples/sql-persistence/injecting-services/SqlPersistence_6/Endpoint/Program.cs

```diff
@@ -13,7 +13,8 @@ static async Task Main()
         var endpointConfiguration = new EndpointConfiguration("Samples.SqlPersistence.InjectingServices");
         endpointConfiguration.EnableInstallers();
 
-        var connectionString = @"Data Source=.\SqlExpress;Initial Catalog=NsbSamplesInjectedServices;Integrated Security=True";
+        // for SqlExpress use Data Source=.\SqlExpress;Initial Catalog=NsbSamplesInjectedServices;Integrated Security=True;Encrypt=false
```

Comment:
Why not go with `sqlexpress` as default? This would increase the chances of `F5` "just working".

Reply:
I have no objections of flipping things around again. I just feel these days with .NET being more and more cross-platform and people embracing docker in all sorts of environments, this change would make it less windows focused. 

Also, when you look at the prerequisites we are pretty detailed about what you need to have "installed" for it to run so I wonder how much of this is really "just F5". I think you have more experience with these samples and their "whereabouts"  so I'd be happy to adjust in any direction.